operator/Makefile

```
@@ -180,7 +180,7 @@ define go-install-tool
 set -e; \
 package=$(2)@$(3) ;\
 echo "Downloading $${package}" ;\
-GOBIN=$(LOCALBIN) go install $${package} ;\
+GOFLAGS='' GOBIN=$(LOCALBIN) go install $${package} ;\
```

Can this solve the problem without overriding all the go flags?

```suggestion
GOBIN=$(LOCALBIN) go install -mod=mod $${package} ;\
```

---

Using `-mod=mod` also worked.